cdap-ui/app/cdap/components/CaskWizards/ArtifactUpload/index.js

```diff
@@ -66,12 +68,16 @@ export default class ArtifactUploadWizard extends Component {
     let subtitle = T.translate('features.Wizard.ArtifactUpload.subtitle');
     let buttonLabel = T.translate('features.Wizard.ArtifactUpload.callToAction');
     let linkLabel = T.translate('features.Wizard.GoToHomePage');
-    this.successInfo.message = `${defaultSuccessMessage} "${name}".`;
-    this.successInfo.subtitle = subtitle;
-    this.successInfo.buttonLabel = buttonLabel;
-    this.successInfo.buttonUrl = `/hydrator/ns/${namespace}/studio`;
-    this.successInfo.linkLabel = linkLabel;
-    this.successInfo.linkUrl = `/cdap/ns/${namespace}`;
+    this.setState({
+      successInfo: {
+        message: `${defaultSuccessMessage} "${name}".`,
+        subtitle,
+        buttonLabel,
+        buttonUrl: `/pipelines/ns/${namespace}/studio`,
```

Comment:
@tonybach We should use `url-generator` utility functions here. We shouldn't be hand coding the url for other webapps.